ci_build/azure_pipelines/mobile-examples-pipeline.yml

```diff
@@ -280,3 +280,45 @@ jobs:
     parameters:
       xcWorkspacePath: 'mobile/examples/super_resolution/ios/ORTSuperResolution/ORTSuperResolution.xcworkspace'
       scheme: 'ORTSuperResolution'
+
+- job: QuestionAnsweringAndroid
+  pool:
+    vmImage: "macOS-11"
+
+  steps:
+  - template: templates/use-python-step.yml
+  - bash: |
+      set -e
+      bash ./prepare_model.sh
+    workingDirectory: 'mobile/examples/question_answering/android'
+    displayName: "Generate model"
+    
+  - script: |
+      python3 ./ci_build/python/run_android_emulator.py \
+        --android-sdk-root ${ANDROID_SDK_ROOT} \
+        --create-avd --system-image "system-images;android-30;google_apis;x86_64" \
+        --start --emulator-extra-args="-partition-size 4096" \
+        --emulator-pid-file $(Build.BinariesDirectory)/emulator.pid
+    displayName: "Start Android emulator"
+  
+  - task: JavaToolInstaller@0
+    displayName: Use jdk 11
+    inputs:
+      versionSpec: "11"
+      jdkArchitectureOption: "x64"
+      jdkSourceOption: "PreInstalled"
+
+  - bash: |
+      set -e
+      chmod +x ./gradlew 
```

Comment:
nit: can we just make the checked in gradlew file executable?
https://git-scm.com/docs/git-update-index#Documentation/git-update-index.txt---chmod-x

Reply:
Sure.Done.